lms/envs/common.py

```diff
@@ -3634,6 +3634,8 @@ def _make_locale_paths(settings):  # pylint: disable=missing-function-docstring
 
 POLICY_CHANGE_GRADES_ROUTING_KEY = 'edx.lms.core.default'
 
+SINGLE_LEARNER_COURSE_REGRADE_ROUTING_KEY = 'edx.lms.core.default'
```

Comment:
why would we not be using this? https://github.com/edx/edx-internal/pull/6911/files#diff-626c5dd75a340caa5f1c53c21ca9b94156b7c0dc0665f4134d03d35940c3d27fR325 Or are we doing that in a new PR after this? I could use some info on how these changes fit together/into a plan moving forward

Reply:
@jansenk are you following the pattern set by other routing keys, eg the `POLICY_CHANGE_GRADES_ROUTING_KEY` for example?